lib/test-worker.js

```
@@ -30,6 +30,20 @@ process.installPrecompilerHook();
 var dependencies = [];
 process.installDependencyTracking(dependencies, testPath);
 
+if (opts.extensions) {
+	opts.extensions.forEach(function (ext) {
+		switch (ext) {
+			case 'ts':
+				var extTs = require('./ext/ts');
+				extTs.register();
+				break;
+			default:
+				// not supported ext?
```

It should throw an useful error here.